### ISSUE TITLE
Guide instructions

### ISSUE THREAD

**Issue by Suncatcher** [Romastyi/VLC-VK-Music#2]
Hi!
Could you, please, provide some guide?)) I installed plugin but cannot notice any visual changes or meny items. How can I access plugin?

**Comment by UnikZ** [comment]
@Suncatcher, after installation, you can see additional menu item ("View" -> "vk.com music player"). Click it. Then pop-up window will be shown.

First of all, you will need to configure plugin (grant access to your VK profile) :
- click "config" button on the bottom toolbar
- click first link to grant access
- copy access_token and user_id query parameters from browser address bar
- past them to particular text fields
- apply changes

Now you can search/play any music from vk.com

**Comment by Suncatcher** [comment]
It seems that plugin was installed correctly, but I still see no items in View menu.
![shot-2015-10-04_16-41-04](https://cloud.githubusercontent.com/assets/6388034/10268203/33c8b4c0-6aba-11e5-86fd-c2464ca35bcd.jpg)

![archlinux-screen](https://cloud.githubusercontent.com/assets/6388034/10268196/f2ca3570-6ab9-11e5-9d0b-e6b20e2e0502.png)

**Comment by UnikZ** [comment]
Hmm.. It is so strange. Is there vkmusic.lua plugin in "Active Extensions" tab?
![screenshot from 2015-10-04 17 31 06](https://cloud.githubusercontent.com/assets/575850/10268362/d956b05a-6abe-11e5-8c5f-3b5f004abfbd.png)

**Comment by Suncatcher** [comment]
No, it is not listed in "Active Extensions". What could that mean? What do you mean under "different plugin versions"?
I have the latest 2.2.1.7 version of VLC.

**Comment by UnikZ** [comment]
Click "Reload plugins" button in "Active Extensions" tab. I think, this should help.

**Comment by Suncatcher** [comment]
Tried already without success. Are you also using the latest version of VLC?

**Comment by UnikZ** [comment]
> Tried already without success.

It is sad.

I'm using VLC 2.2.0 from Ubuntu repository. How did you install the plugin?

**Comment by Suncatcher** [comment]
Maybe this is the reason - version incompatibility?

> How did you install the plugin?

This is interesting question. I think I copied it to `~/.local/vlc/lua/extension`, if I not mistaken. Previosuly I tried to copy it to `/usr/lib/vlc/lua/extension` but it didn't work. Maybe I installed it in a wrong way?

**Comment by UnikZ** [comment]
I copied to `~/.local/share/vlc/lua/extensions/` path.

I have tried to install the same VLC version, but now I have the latest 3.0.0 git master version and plugin still works fine.

**Comment by Suncatcher** [comment]
Tried to reinstall several times, tried to copy to root `.local` directory and user `.local`. No luck. 
I cannot make it to be showed in active plugins. Are there any additional steps for this needed?

**Comment by UnikZ** [comment]
I have no more ideas, what can be wrong. Fast google search shows several issues with vlc 2.1.x and .lua extensions. So, can you downgrade VLC to stable 2.0.x or upgrade to unstable 3.0.x version?

**Comment by Suncatcher** [comment]
Finally I was able to install plugin on another distro and another VLC, and, finally, it apperared in menu.
But after I press the menu item nothing happens. What is wrong?

![virtualbox_bodhi_29_11_2015_00_18_23](https://cloud.githubusercontent.com/assets/6388034/11453960/48e6984a-962f-11e5-8486-e2562bccf2ca.png)
